entrypoint.sh

```diff
@@ -52,24 +52,22 @@ run_test_if_needed() {
 }
 
 create_prefix() {
-  FLUTTER_PREFIX=""
-  PUB_PREFIX="pub"
```

Comment:
Is there a difference between `dart pub` and `flutter pub`? It would be simpler to have `PUB_PREFIX="dart pub"` and use that everywhere

Reply:
@CaseyHillers 

I think you said right. https://github.com/flutter/flutter/issues/88507
I will fix the next release.